### PR TITLE
update file version for VS vNext

### PR DESCRIPTION
**Pull request by faxue-msft**
@gregg-miskelly @chuckries

### CODE REVIEW

File: build/version.settings.targets [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>    
     <!--SxS: These three properties should be changed at the start of a new version, VersionZeroYear should be the year
     before the start of the project. When updating the version, also update MIEngine\metadata.json.-->
-    <MajorVersion>14</MajorVersion>
+    <MajorVersion>15</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <VersionZeroYear>2013</VersionZeroYear>
```
Comment (gregg-miskelly) [inline build/version.settings.targets]: I would change 'VersionZeroYear' to 2015 when we do this.

**Comment by gregg-miskelly** [comment]
I would hold off on this for another month. We are planning to ship one more dev14 version, and it might be confusing that the version number in Dev15.

**Comment by gregg-miskelly** [comment]
@caslan what do you think?

File: build/version.settings.targets (hunk 1 above)
Comment (faxue-msft) [inline build/version.settings.targets]: I thought it's the start time for miengine project, isn't it?

**Comment by faxue-msft** [comment]
@gregg-miskelly
Just to be clear, do you mean shipping MIEngine as dev14 update 4 for coreclr debugging?

**Comment by caslan** [comment]
As an OOB. We are planning to release by the end of September for SSH support for VC++ Linux extension.

File: build/version.settings.targets (hunk 1 above)
Comment (gregg-miskelly) [inline build/version.settings.targets]: It says what real year should have a '1' in the year part of the version number (MIEngine uses build numbers of YMMDD). But the max number of years which this scheme can accept is 6, so when we rev the major year, we also want to update the VersionZeroYear.

**Comment by faxue-msft** [comment]
Per our discussion, we'll stick with the version number 14

**Pull request closed without merging**